test/ui-test/src/screens/StatusWelcomeScreen.py

```diff
@@ -84,7 +85,8 @@ def input_username_and_password_and_finalize_sign_up(self, username: str, passwo
         self.input_password(password)
         click_obj_by_name(SignUpComponents.FINALIZE_PASSWORD_STEP.value)
 
-        click_obj_by_name(SignUpComponents.PASSWORD_PREFERENCE.value)
+        if sys.platform == "Darwin":
```

Comment:
Testing it, my system expects "darwin" instead of "Darwin". Please change the capital letter!

Reply:
updated